src/common/api_strings.hpp

```diff
@@ -52,6 +52,7 @@
 #define OTBR_DHCP6_PD_STATE_NAME_DISABLED "disabled"
 #define OTBR_DHCP6_PD_STATE_NAME_STOPPED "stopped"
 #define OTBR_DHCP6_PD_STATE_NAME_RUNNING "running"
+#define OTBR_DHCP6_PD_STATE_NAME_IDLE "idle"
```

Comment:
add doxgen?

Reply:
I think this symbol is not exposed? And we have no doxgen within this document?